test/test_rospy/test/rostest/test_basic_services.py

```diff
@@ -213,6 +214,28 @@ def test_String_String(self):
             resp_req_kwds = self._test_req_kwds(name, Cls, {'str': String('FOO'), 'str2': Val('bar')})
             for resp in [resp_req, resp_req_naked, resp_req_kwds]:
                 self.assertEquals('FOObar', resp.str.data)
+
+    def test_String_String_unicode(self):
+        from std_msgs.msg import String
+        from test_rospy.srv import StringString, StringStringRequest
+        from test_rospy.msg import Val
+        import sys
+        Cls = StringString
+        Req = StringStringRequest
+
+        for name in [STRING_CAT_SERVICE_NAKED, STRING_CAT_SERVICE_WRAPPED]:
+            if sys.version < '3':
```

Comment:
I'm not sure the version check is even required. The `u` prefix syntax is valid in both Python 2 and 3. In 2, it produces a `unicode`, and in 3, a `str`. So I think the first block would be adequate for both versions.